shardingsphere-infra/shardingsphere-infra-common/src/main/java/org/apache/shardingsphere/infra/instance/InstanceContext.java

```diff
@@ -52,6 +52,9 @@ public InstanceContext(final ComputeNodeInstance instance, final WorkerIdGenerat
         this.workerIdGenerator = workerIdGenerator;
         this.modeConfiguration = modeConfiguration;
         this.lockContext = lockContext;
+        if (!isCluster()) {
```

Comment:
for standalone mode, DistSQL just control the current instance, So there is no need to add instance.

Reply:
Maybe in standalone mode, there is no need to check whether the ComputerNodeInstance is consistent?